packages/web-embed-api/src/index.ts

```diff
@@ -122,6 +123,14 @@ const FormsortWebEmbed = (
     unauthorized: [],
   };
 
+  const onStyleSetRequest = () => {
+    sendMessage({
+      type: WebEmbedMessage.EMBED_STYLE_SET_RESPONSE_MSG,
+      // @ts-ignore - Intentionally omitted from IFormsortWebEmbedConfig
+      payload: config.styleSet || undefined,
```

Comment:
I would prefer that we define the type on config and add a warning to it. Something like
```ts
  /**
   * For Formsort internal use only
   */
  styleSet?: Record<string, unknown>;
```

Reply:
Updated to include type! Removing `@ts-ignore` also helped me discover an issue with the response 😅